typescript/examples/by-use-case/evm-send-and-receive-tokens/index.ts

```diff
@@ -22,17 +19,13 @@ const account = privateKeyToAccount(process.env.WALLET_PRIVATE_KEY as `0x${strin
 const walletClient = createWalletClient({
     account: account,
     transport: http(process.env.RPC_PROVIDER_URL),
-    chain: baseSepolia,
+    chain: sepolia,
```

Comment:
why this change?

Reply:
No reason, wanted to test on different networks.